### ISSUE TITLE
[BUG] Load showing incorrectly after latest update

### ISSUE THREAD

**Issue by MattJeanes** [flixlix/power-flow-card-plus#235]
**Describe the bug**

After the latest update(s) to the card, it seems that there is an issue showing load and it is way off, in this particular case showing 3.2kW when it should have been 1.44kW (or thereabouts).

I know it is possible to configure it to use the actual given load entity but it should be pretty accurate with it off.

**To Reproduce**

Minimal config to repro:
```yaml
type: custom:power-flow-card-plus
entities:
  battery:
    entity:
      consumption: sensor.growatt_battery_discharging_actual
      production: sensor.growatt_battery_charging_actual
  grid:
    entity:
      consumption: sensor.growatt_import_from_grid_actual
      production: sensor.growatt_export_to_grid_actual
  solar:
    entity: sensor.growatt_input_kilowatt_actual
  home:
    entity: sensor.growatt_local_load_consumption_actual
```

Entity values at time of issue:
```yaml
sensor.growatt_battery_discharging_actual: 0W
sensor.growatt_battery_charging_actual: 1.87kW
sensor.growatt_import_from_grid_actual: 0W
sensor.growatt_export_to_grid_actual: 9W
sensor.growatt_input_kilowatt_actual: 3.26kW
sensor.growatt_local_load_consumption_actual: 1.44kW
```

**Expected behavior**
Load to closely match the local_load_consumption_actual

**Screenshots**

![image](https://github.com/flixlix/energy-flow-card-plus/assets/2363642/f8c05853-0de5-4f78-aa4a-2ad008a6c336)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome 114.0.5735.90
 - Versions:
    - This component: v0.1.4.1
    - Home Assistant 2023.5.4
    - Supervisor 2023.04.1
    - Operating System 10.2
    - Frontend 20230503.3 - latest

**Additional context**

This issue appeared to start after updating the addon, it was not present in version v0.1.3.

Adding `override_state` to the home entity works around the issue.

**Please make sure all of this applies and check the checkboxes, by writing "X" inside of them.**

- [x] I understand that this is a channel for reporting bugs, not a support forum (If you need help configuring the card, go to: [Discussions](https://github.com/flixlix/power-flow-card-plus/discussions)).

- [x] I have the latest version of the card installed.

- [x] I have cleared my browser cache **after** installing, updating and configuring the card.

**Comment by stepir** [comment]
confirmed I had to rollback wot 1.3 to solve

**Comment by flixlix** [comment]
Duplicate of #231 and possibly #227, will try to fix as soon as possible, but for now stay with 0.1.3

**Comment by MattJeanes** [comment]
Ah sorry, didn't see the existing issue cause I originally started writing this on the energy-flow-card repo and checked for existing issues in there before realising it was power-flow-card. Thanks for looking at it anyway 🙂

**Comment by flixlix** [comment]
Just released v0.1.5 :)
